Components/Navigation/Navigation.css

```diff
@@ -0,0 +1,48 @@
+.navbar-container{
+    background-color: var(--black-font-color);
+    color: var(--white-font-color);
+}
+
+.nav-style{
+    text-align: center;
+    padding: 0;
+    cursor: pointer;
+}
+
+
+.header-container{
+    height: 400px;
+    overflow-y: scroll;
+    width: 80%;
+    margin: 0 auto;
+    border: 1px 0.9 solid var(--secondary-alert) ;
```

Comment:
This Property will not work as the border only takes single value for the width.
_border: 1px solid var(--secondary-alert) ;_  this will work.

Reply:
yeah it didnt work forgot to omit that